packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SubjectGroupViewer/components/SGVGridCell/SGVGridCell.js

```diff
@@ -54,6 +79,25 @@ function SGVGridCell (props) {
   const imageY = (cellHeight - imageHeight) / 2
 
   const clipPathID = `subjectGroupViewer-clipPath-${index}`
+  
+  function toggleCellAnnotation () {
+    if (!annotationMode || !annotation?.value) return
+
+    const toggledValue = !checked
+    setChecked(toggledValue)
+
+    const annotationValue = annotation?.value?.slice() || []
+    const hasCellIndex = annotationValue.includes(index)
+    
+    if (hasCellIndex && !toggledValue) {  // Remove cell index from annotation values
+      const indexInValue = annotationValue.indexOf(index)
+      annotationValue.splice(indexInValue, 1)
+    } else if (!hasCellIndex && toggledValue) {  // Add cell index to annotation values
+      annotationValue.push(index)
+    }
+    
+    if (annotation?.update) annotation.update(annotationValue)
```

Comment:
If `annotation` is undefined, then we can't store a new value here. That makes logical sense, but creates a case where a task is rendered but never saves annotations. I think it would be better to make `annotation` required, not optional.

Reply:
Ah! That scenario has already been accounted for - what if a Workflow wants to use the Subject Group Viewer but not actually use a Subject Group Comparison Task? (i.e. no `annotation`)

Example: https://local.zooniverse.org:8080/?env=staging&project=1900&workflow=3402 (`workflow.config.subject_viewer = 'subjectGroup'; T0.type = 'single'`)

<img width="907" alt="Screen Shot 2020-11-18 at 21 06 52" src="https://user-images.githubusercontent.com/13952701/99730338-2cdb9500-2ab4-11eb-9aaf-2a0a7b8221b1.png">
_Example of a Subject Group Viewer with a Single Choice Task_

In this scenario, the SGVGridCell continues to render individual images and allow users to pan the image (mouse/keyboard), without responding to any click-to-select-cell (or keyboard equivalent).